di_website/templates/datasection/data_section_page.html

```diff
@@ -0,0 +1,158 @@
+{% extends "layouts/default.html" %}
+{% load navigation_tags static wagtailcore_tags wagtailimages_tags %}
+
+{% block hero %}
+  {% include 'includes/heroes/hero.html' %}
+{% endblock hero %}
+
+{% block body %}
+
+<section class="section">
+  <div class="row row--narrow">
+    <div class="l-sidebar">
+      {% comment %} TODO:: Add more about section here {% endcomment %}
```

Comment:
Fun fact, you can also do single line comments like
`{# this is a comment #}`

Reply:
This has cleaned up the comments section, thanks :)